packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -459,10 +485,10 @@ class Auth {
             id_token: this.get('id_token'),
             idp_access_token: this.get('idp_access_token'),
             refresh_token: this.get('refresh_token_registered') || this.get('refresh_token_guest'),
-            token_type: this.get('token_type'),
```

Comment:
Does this mean we are not using `DATA_MAP['token_type']` any more?

Reply:
The change was made by Joel so I'm not sure about the reason. 
Looking into the `commerce-sdk-isomorphic@4.0.0` I see always returns 'Bearer' as the token_type, and thus ts complains if you don't provide 'Bearer'. If commerce-sdk-isomorphic ever changes that, it will be a breaking change, so if we want to keep `DATA_MAP['token_type']` for consistency, we can do it vi assertion instead of hardcoding it. I made the change.